bika/health/static/js/bika.health.analysisrequest.add.js

```diff
@@ -92,7 +93,7 @@ function HealthAnalysisRequestAddView() {
                 type: 'POST',
                 data: {
                     '_authenticator': $('input[name="_authenticator"]').val(),
-                    'ClientPatientID': id
+                    'PatientUID': id
```

Comment:
Well done, but if we are using the patient UID, the function description should be updated as well.